vivarium_public_health/risks/base_risk.py

```diff
@@ -43,8 +36,8 @@ def setup(self, builder):
         builder.event.register_listener('time_step__prepare', self.update_exposure, priority=8)
 
     def load_population_columns(self, pop_data):
-        population = self.population_view.get(pop_data.index, omit_missing_columns=True)
-        propensities = pd.Series(self.propensity_function(population, self._risk),
+        population = self.population_view.get(pop_data.index, omit_missing_columns=True )
```

Comment:
When I remove the `omit_missing_columnns=True`, actually the current `test_base_risk` fails for all cases